pypeit/ginga.py

```diff
@@ -231,7 +231,7 @@ def show_slits(viewer, ch, lord_in, rord_in, slit_ids = None, rotate=False, pste
     ch
     lord_in : ndarray
     rord_in : ndarray
-    slit_ids : list of int
+    slit_ids  (list of int):
```

Comment:
So the type is list.  Specify in the description that it should be list of ints, and then check that this is true in the code.